pkg/dao/repository_configs.go

```diff
@@ -422,9 +423,11 @@ func (r repositoryConfigDaoImpl) filteredDbForList(OrgID string, filteredDB *gor
 			Where("name ILIKE ? OR url ILIKE ?", containsSearch, containsSearch)
 	}
 
-	if !config.Get().Features.NewRepositoryFiltering.Enabled && filterData.Origin == "" {
+	// Default origin to "external" if not specified
+	if filterData.Origin == "" {
 		filterData.Origin = config.OriginExternal
```

Comment:
same comment here, we don't want to default to originExternal now, as we want the feature to be 'turned on' permanently